### PR TITLE
Docs tidy

### PR DESCRIPTION
**Pull request by faph**
I've restructured and tidied the docs a little bit. The main thing is that I have seperated the packager manager versus package builder docs as much as possible. The intro is also a bit less techy with a focus on explain the ecosystem with references to the examples/tutorials and command reference.

I can probably still tidy this further, add some cross-references, etc. 

Let me know what you think.

### CODE REVIEW

File: docs/source/build.rst [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .. _build:
 
-conda build Recipe Reference
+Conda build recipe reference
```
Comment (asmeurer) [inline docs/source/build.rst]: Why this change? The titles should be in title case.

File: docs/source/conf.py [hunk 2]
```diff
@@ -54,7 +54,7 @@
 
 # General information about the project.
 project = u'Conda'
-copyright = u'2014, Continuum Analytics'
+copyright = u'2014‒2015, Continuum Analytics'
```
Comment (asmeurer) [inline docs/source/conf.py]: Let's just use the current year.

File: docs/source/intro.rst [hunk 3]
```diff
@@ -2,189 +2,145 @@
...
 
-To view all available packages, you can use ``conda search``.  See the :ref:`search command examples <search_example>` for more information.
+.. _meta_package:
 
-Once a conda package has been downloaded, it is said to
-be `locally available`.
-A locally available package that has been linked into an conda environment
-is said to be `linked`.
-Conversely, unlinking a package from an environment causes it to be `unlinked`.
+Meta-Packages
```
Comment (asmeurer) [inline docs/source/intro.rst]: We usually spell this without the dash

Comment (asmeurer) [inline docs/source/intro.rst]: Ah I see this was already here already.

File: docs/source/build.rst (hunk 1 above)
Comment (faph) [inline docs/source/build.rst]: I think there is a little bit of inconsistency between the pages. Happy to change everything to title case.

File: docs/source/conf.py (hunk 2 above)
Comment (faph) [inline docs/source/conf.py]: ok

File: docs/source/intro.rst [hunk 4]
```diff
@@ -2,189 +2,145 @@
...
 
-.. _activated:
-.. index::
-    pair: terminology; activated
+    ``gevent-websocket-0.3.6-py27_0.tar.bz2``
 
-.. _deactivated:
-.. index::
-    pair: terminology; deactivated
+    <package-name>-<version>-<build_string>.tar.bz2
```
Comment (asmeurer) [inline docs/source/intro.rst]: This could be formatted better.

Comment (asmeurer) [inline docs/source/intro.rst]: We also have that image, which explains this pretty well.

File: docs/source/intro.rst (hunk 3 above)
Comment (faph) [inline docs/source/intro.rst]: sure. Will check throughout.

File: docs/source/pip-comparison.rst [hunk 5]
```diff
@@ -0,0 +1,26 @@
+===================
+Conda for pip users
```
Comment (asmeurer) [inline docs/source/pip-comparison.rst]: Let's keep the name "Rosetta Stone"

Comment (faph) [inline docs/source/pip-comparison.rst]: I have no idea what "Rosetta Stone" means. I discovered the section very late and had no idea what to expect under that title. Not sure if this is an international thing, otherwise I would suggest to use as plain English as it gets.

**Comment by faph** [comment]
Build failure is due to advanced.rst no longer being referenced. It's empty so I will remove it?

File: docs/source/pip-comparison.rst (hunk 5 above)
Comment (asmeurer) [inline docs/source/pip-comparison.rst]: I had thought the [Rosetta Stone](https://en.wikipedia.org/wiki/Rosetta_Stone) was pretty famous. Anyway it's influenced by https://github.com/sympy/sympy/wiki/Git-hg-rosetta-stone.

**Comment by asmeurer** [comment]
Yes, it looks like that page isn't very useful.

File: docs/source/intro.rst (hunk 4 above)
Comment (faph) [inline docs/source/intro.rst]: Agree but for intro it felt overkill to me. Or should we remove any reference to a filename at this point?

Comment (asmeurer) [inline docs/source/intro.rst]: Yeah for the intro page the filename is too specific of a detail I'd say.

**Comment by asmeurer** [comment]
@electronwill @kerrywatson1 you may want to run through this to see if anything is still relevant. I am closing this because most of it can't be applied any more.
